service/history/ndc/workflow_resetter.go

```diff
@@ -654,9 +661,10 @@ func (r *workflowResetterImpl) reapplyContinueAsNewWorkflowEvents(
 	return lastVisitedRunID, nil
 }
 
-func (r *workflowResetterImpl) reapplyWorkflowEvents(
+func (r *workflowResetterImpl) reapplyEventsFromBranch(
 	ctx context.Context,
 	mutableState workflow.MutableState,
+	currentUpdateRegistry update.Registry,
```

Comment:
this is not used and can be removed?

Reply:
Thanks! Yes that was left over from a refactoring.